<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapListWidget</class>
 <widget class="QWidget" name="MapListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_ml_gameMode">
       <property name="text">
        <string>Gamemode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ml_gameMode"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_ml_availableSelectedMapImage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_ml_availableSelectedMapText">
         <property name="text">
          <string>Preview of selected map</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_ml_currentMapText">
         <property name="text">
          <string>Current map</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_ml_currentMapValue">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_ml_currentMapImage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_ml_currentSelectedMapImage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_ml_currentSelectedMapText">
         <property name="text">
          <string>Preview of selected map</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="DragDropTreeWidget" name="treeWidget_ml_current">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gamemode</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rounds</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="DragDropTreeWidget" name="treeWidget_ml_available">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gamemode</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_ml_load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ml_save">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/save.png</normaloff>:/frostbite/icons/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ml_clear">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/clear.png</normaloff>:/frostbite/icons/clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ml_add">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/add.png</normaloff>:/frostbite/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ml_rounds">
       <property name="text">
        <string>Rounds</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_ml_rounds">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ml_remove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../frostbite/Frostbite.qrc">
         <normaloff>:/frostbite/icons/remove.png</normaloff>:/frostbite/icons/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragDropTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>DragDropTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../frostbite/Frostbite.qrc"/>
 </resources>
 <connections/>
</ui>
